### ISSUE TITLE
Peptide length limitation

### ISSUE THREAD

**Issue by DivyaRavichandar** [artyomovlab/hmmhc#4]
Currently this code works for peptides in 12 -24 AA range. it appears this in enforced for the percentile calculation. Would removing this limitation still allow for the log odd calcualtions?

**Comment by ikizhvatov** [comment]
Yes, technically log odds is computable also outside of this range.

**Comment by DivyaRavichandar** [comment]
Thanks. Since the tool currently is restricted to this range. IF I were to use this to calcualte log odds for longer peptides (say 10 - 40 aa).

Would I simply take out the length checks on line 86 in the hmmhc.py script?

**Comment by ikizhvatov** [comment]
Yes, it  should work. Beware though that for the lengths outside of the range the predictions may be less reliable, as the training set contained mostly lengths 12-24.
